plugin/trino-fission-functions/src/main/java/io/trino/plugin/fission/FissionFunctionPlugin.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.fission;
+import com.google.common.collect.ImmutableSet;
+import io.trino.spi.Plugin;
+
+import java.util.Set;
+
+public class FissionFunctionPlugin
```

Comment:
Javadoc? I keep saying this just because I'm seeing public things. If this is a standard set of things required for Trino functions, maybe we can go light on the javadoc.